README.md

```diff
@@ -7,7 +7,12 @@ Thinkst Applied Research
 Overview
 ----------
 
-OpenCanary is a daemon that runs several canary versions of services that alerts when a service is (ab)used.
+In essence, OpenCanary creates a network honeypot allowing you to catch hackers before they fully compromise your systems. As a technical definition, OpenCanary is a daemon that runs several canary versions of services that alerts when a service is (ab)used. 
+
+Features
+----------
+
+* Recieve email alerts as soon as potential threats are detected, highlighting the threat source IP address and where the breach may have taken place. 
```

Comment:
spelling correction here: `receive`
I figure if we creating a `Features` section, we should add a few other alerting mechanisms too then. Webhooks, syslog etc. 
We can also add the services that are offered. What do you think?

Reply:
Sounds good. Feel free to add those sections.

Reply:
Cool, let me know once the spelling correction is made